### ISSUE TITLE
Order emails should no longer catch errors

### ISSUE THREAD

**Issue by AugustMiller** [craftcms/commerce#1379]
Now that order emails are sent via the Queue, failure can be tracked in other ways (aside from logging the exception messages).

Debugging would be _much_ simpler if you could visualize what emails failed to send, fix the error, and then re-try, rather than failing quietly… but still completing in the queue!

Basically, this chunk of logic to swallow template processing errors feels a bit out of place, in the new model, and could probably just be allowed to blow up and report exceptions to the Queue: https://github.com/craftcms/commerce/blob/develop/src/services/Emails.php#L430-L849

In other words—now that we have a means to record and recover from failures, we should probably do that, rather than hiding rendering/deliverability issues.

**Comment by lukeholder** [comment]
Great idea. Will do.

**Comment by gojira32** [comment]
Any progress on this. We just got caught out with this.

> Debugging would be much simpler if you could visualize what emails failed to send, fix the error, and then re-try, rather than failing quietly… but still completing in the queue!

This would have been super handy...

**Comment by lukeholder** [comment]
Thanks for your PR @AugustMiller - you were right though, it was breaking change to the `sendEmail()` expecting to only return a bool.

I have gone ahead made an alternative fix in 383f53b0db4ede34d4ccaea6154483cdaf6b3ac2

Will be in the next release. Thanks!
